catalog/catalog-app/pom.xml

```diff
@@ -327,6 +327,11 @@
         </dependency>
         <dependency>
             <groupId>ddf.catalog.security</groupId>
+            <artifactId>catalog-security-operationplugin</artifactId>
```

Comment:
Not on this line, but should fix while in there:
1- org.codice.thirdparty jts dependency is duplicated (~143 and ~313)
2- file could use a reformat (maybe in separate commit?)